module/applications/actor/base-sheet.mjs

```diff
@@ -860,7 +861,9 @@ export default class ActorSheet5e extends ActorSheet {
     const value = input.value;
     if ( ["+", "-"].includes(value[0]) ) {
       const delta = parseFloat(value);
-      input.value = Number(foundry.utils.getProperty(this.actor, input.name)) + delta;
+      const item = this.actor.items.get(input.closest("[data-item-id]")?.dataset.itemId);
+      if ( !item ) input.value = Number(foundry.utils.getProperty(this.actor, input.name)) + delta;
+      else input.value = Number(foundry.utils.getProperty(item, input.dataset.name)) + delta;
```

Comment:
Prefer to switch these so that the `else` statement does not read like a double-negative:
```suggestion
      if ( item ) input.value = Number(foundry.utils.getProperty(item, input.dataset.name)) + delta;
      else input.value = Number(foundry.utils.getProperty(this.actor, input.name)) + delta;
```